vhost-device-sound/Cargo.toml

```diff
@@ -36,5 +36,5 @@ pw = { package = "pipewire", version = "0.8", optional = true }
 [dev-dependencies]
 rstest = "0.18.2"
 tempfile = "3.9"
-virtio-queue = { version = "0.11", features = ["test-utils"] }
+virtio-queue = { version = "0.11", git = "https://github.com/rust-vmm/vm-virtio.git", features = ["test-utils"]}
```

Comment:
I understand that this is still provisional, but why does this commit come last? I would have expected it to come first in history.

Reply:
I will move it to the beginning.